src/sider/arbeidsuforhet/avslag/AvslagForm.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { useSendVurderingArbeidsuforhet } from "@/data/arbeidsuforhet/useSendVurderingArbeidsuforhet";
+import { useArbeidsuforhetVurderingDocument } from "@/hooks/arbeidsuforhet/useArbeidsuforhetVurderingDocument";
+import { FormProvider, useForm } from "react-hook-form";
+import {
+  VurderingRequestDTO,
+  VurderingType,
+} from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
+import { BodyShort, Box, Button, Heading, Textarea } from "@navikt/ds-react";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { ButtonRow } from "@/components/Layout";
+import { Forhandsvisning } from "@/components/Forhandsvisning";
+import { Link } from "react-router-dom";
+import { arbeidsuforhetPath } from "@/routers/AppRouter";
+import { AvslagDatePicker } from "@/sider/arbeidsuforhet/avslag/AvslagDatePicker";
+import dayjs from "dayjs";
+
+const texts = {
+  title: "Skriv innstilling til NAY",
+  begrunnelseLabel: "Innstilling om avslag (obligatorisk)",
+  info1:
+    "Skriv kort hvilke opplysninger som ligger til grunn for avslaget, samt din vurdering av hvorfor vilkåret ikke er oppfylt og vurdering av eventuelle nye opplysninger.",
+  info2:
+    "Hvis du har vurdert ordningen friskmelding til arbeidsformidling: skriv hvorfor ordningen ikke er aktuell og legg inn henvisning til §8-5.",
+  afterSendInfo: {
+    header: "Videre må du huske å:",
+    tasks: [
+      "Sende oppgave til NAY i Gosys.",
+      "Gi beskjed om avslag til ny saksbehandlingsløsning via Stoppknappen under fanen Sykmeldinger i Modia.",
+    ],
+  },
+  buttonDescription:
+    "Når du trykker “Gi avslag” blir innstillingen journalført og lagt til Gosys. ",
+  forhandsvisningLabel: "Forhåndsvis vurderingen",
+  missingBegrunnelse: "Vennligst angi begrunnelse",
+  sendVarselButtonText: "Gi avslag",
+  avbrytButton: "Avbryt",
+};
+
+const defaultValues = { begrunnelse: "", fom: undefined };
+const begrunnelseMaxLength = 1000;
```

Comment:
Holder med 1000 eller skulle vi hatt 5000 her også?

Reply:
Hmm, vet ikke! Jeg kan høre med Stine 👍

Reply:
Stine tenker 5000, så da går vi for det 🎉